compiler-rt/test/asan/TestCases/Darwin/interface_symbols_darwin.cpp

```diff
@@ -35,6 +35,9 @@
 // RUN:  %p/../../../../lib/sanitizer_common/sanitizer_coverage_interface.inc     \
 // RUN:  | grep -e "INTERFACE_\(WEAK_\)\?FUNCTION"                                \
 // RUN:  | grep -v "__sanitizer_weak_hook"                                        \
+// RUN:  | grep -v "__sanitizer_override_function"                                \
```

Comment:
can we move related changes into a separate patch

Reply:
It would cause test failures on linux/darwin. These tests grep the interface files for "expected" imports and do not run the preprocessor on them, so the new interface symbols on windows need to be excluded.